bundles/org.openhab.core.addon/.classpath

```diff
@@ -24,16 +24,16 @@
 			<attribute name="annotationpath" value="target/dependency"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" output="target/test-classes" path="src/test/java">
```

Comment:
What happened here?

Reply:
Umm. Not sure. I think Eclipse did it automagically when I wrote a new JUnit test.

Reply:
Could you simply revert the changes so that this PR leaves the file untouched?